contracts/CrosschainLockLP.sol

```diff
@@ -0,0 +1,74 @@
+//SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+interface IERC20 {
```

Comment:
Maybe use imports here for IERC20?

Reply:
Certainly, when the number of contracts and the scale of duplication grows, will refactor for imports. For now it's easier to use Remix this way, less stuff to copy around.